tests/test_mirror_port_erspan.py

```diff
@@ -0,0 +1,951 @@
+# This test suite covers the functionality of mirror feature in SwSS
```

Comment:
I think the overall test logic/flow is good, I would just ask if you could use some of the new library functions we have over in `dvslib/dvs_database` and `dvslib/dvs_common` for interacting with the databases. Basically, there are a bunch of functions you can use to poll the database rather than using `time.sleep(1)` throughout. This makes the PR tests a lot more stable and it also speeds up the execution by quite a bit.

There are examples in `test_acl`, `test_vlan`, `test_dpb...`, and `test_sub_port_intf` to name a few. There are also docstrings describing how the methods work as well. I'm also happy to answer any questions if you run into issues. :)

Reply:
Thanks a lot for this suggestion. I have taken care this.
Code looks clean and much faster after this change.